presto-main/src/main/java/com/facebook/presto/cost/JoinStatsRule.java

```diff
@@ -21,14 +21,17 @@
 import com.facebook.presto.sql.planner.plan.JoinNode;
 import com.facebook.presto.sql.planner.plan.PlanNode;
 import com.facebook.presto.sql.tree.ComparisonExpression;
-import com.facebook.presto.sql.tree.ComparisonExpressionType;
 import com.facebook.presto.sql.tree.Expression;
 import com.google.common.collect.ImmutableMap;
 
+import java.util.List;
```

Comment:
Just treat this commit as a commit whic introduces `JoinStatsRule` - mark it in commit message so I will squash the two together